website/tokens/handlers.py

```diff
@@ -0,0 +1,88 @@
+import httplib as http
+
+from modularodm import Q
+
+from framework.auth.decorators import must_be_logged_in
+from framework.exceptions import HTTPError, PermissionsError
+from framework import status
+from framework.flask import redirect
+
+from website.tokens.exceptions import UnsupportedSanctionHandlerKind, TokenError
+
+def registration_approval_handler(action, registration, registered_from):
+    status.push_status_message({
+        'approve': 'Your Registration approval has been accepted.',
+        'reject': 'Your disapproval has been accepted and the registration has been cancelled.',
+    }[action], kind='success', trust=False)
+    if action == 'approve':
+        return redirect(registration.web_url_for('view_project'))
+    else:
+        return redirect(registered_from.web_url_for('view_project'))
+
+def embargo_handler(action, registration, registered_from):
+    status.push_status_message({
+        'approve': 'Your Embargo approval has been accepted.',
```

Comment:
Keep casing of "embargo"/"registration"/etc. consistent. I don't see a reason to uppercase them.